### ISSUE TITLE
Add a setting page

### ISSUE THREAD

**Issue by mmazzarolo** [mmazzarolo/chrome-another-tab#2]
Creating a setting page would allow us to build more toggable features without worrying about where to put the toggle itself. 
E.g.: hide/show header, disable animations, etc...

**Comment by dkozickis** [comment]
That looks easy enough with `storage.sync` Chrome API. Any UI guidelines?

**Comment by mmazzarolo** [comment]
> That looks easy enough with storage.sync Chrome API. Any UI guidelines?

Well, since it was already discussed in the #5 , what about something like Momentum? 

<img width="1247" alt="screenshot 2019-01-15 14 47 03" src="https://user-images.githubusercontent.com/9536354/51184450-99901200-18d4-11e9-8ab1-7c34ee33e7b8.png">

We can start easier, maybe by placing a "cog" icon in the header and then showing the setting at the center (covering the bookmarks).  
Let me know if I can help on it. 

P.S.: This weekend I'll probably tackle #6 , try to move the repo to an organization (I can add you as a collaborator if you want) and release an update.

**Comment by dkozickis** [comment]
Right, let me see what can I come up with. I'd definitely be honored to be a Collaborator on this one :+1: I'll try to live up to the role too. Between debugging my two sons and main work...

**Comment by mmazzarolo** [comment]
> I'll try to live up to the role too. Between debugging my two sons and main work...

Don't worry, it's just a way to not be the only one able to approve/check PR. 
You can stop contributing whenever you want 👌 

I'm working on it on my free time too but I don't have children yet so you'll probably be way more busy than me 😁

**Comment by dkozickis** [comment]
Hey! I'm still "on it", just didn't have time to finish over the weekend. Opened up the code, refactored here, refactored there, in the end didn't have time to introduce new functionality :D I'm planning on writing own modal, but if you want me just to use `react-modal` (it will be 100% accessible then, as opposed to my, possibly, less accessible code). LMK!

**Comment by mmazzarolo** [comment]
@dkozickis no worries! 
Please notice that I had to do a major refactor to implement the bookmarks sorting.

**Comment by dkozickis** [comment]
Looks like bookmarks ordering went a bit kawabanga again. I'm having a week in between jobs this week, I'll try to attend to this one and "that" one.

**Comment by mmazzarolo** [comment]
> Looks like bookmarks ordering went a bit kawabanga again. I'm having a week in between jobs this week, I'll try to attend to this one and "that" one.  

🤔 what do you mean? 
From my testing now both the bookmarks and folders order is working (you might wanna build the extension to make sure the bookmark sorting is working, the fixture might not be 100% correct).
Let me know if you see any issue and I'll try to address that! 👍

**Comment by ONeill45** [comment]
@mmazzarolo @dkozickis is this still in development? If so happy to help out on it if needed.

**Comment by mmazzarolo** [comment]
Don’t know, @dkozickis ?

**Comment by dkozickis** [comment]
@mmazzarolo I'm sorry, I can't find time (or collect my mind) to properly sit down with this.
@ONeill45 please go ahead if you wish 👍

**Comment by ONeill45** [comment]
@dkozickis @mmazzarolo great, i'll start in on this.

**Comment by mmazzarolo** [comment]
@ONeill45 awesome! I'd suggest to work on it gradually sto that we can help you out with it and add feedback on both the UI and the code 👍

**Comment by ONeill45** [comment]
@mmazzarolo Agreed! Gonna start with getting the modal set up and we can go from there.

**Comment by ONeill45** [comment]
@mmazzarolo Here's where I'm at thus far. Added in a styled modal using a wrapped [react-modal](https://github.com/reactjs/react-modal) component along with styled components. Here's a small demo:

![2019-03-05 12 20 34](https://user-images.githubusercontent.com/13375042/53824127-42f09d00-3f41-11e9-9aff-ffc20e99a026.gif)

I mapped the modal to the theme as well and added "modalBackground" so the modal stays consistent with the theme. I used the colors show for demonstration purposes only.

What are your thoughts so far? And any suggestions for the best color to use on each theme? Color has never been my strong suit.

**Comment by mmazzarolo** [comment]
@ONeill45 if you want you can submit a PR (just write "WIP" in the title) so that I can add comments in the code as well.  

About the color... that's tricky, because we might wanna use a variation of the current theme color and work on its opacity.  Maybe we can start with just a dark one (like in the screenshot at the start of the thread)? What do you think?

And thanks! 🙏

**Comment by ONeill45** [comment]
@mmazzarolo will do! 

I actually took another look at Momentum and noticed the opacity so I think that's a great start. I'll adjust before I submit the PR
